modular-docs-manual/content/topics/module_guidelines-assembly.adoc

```diff
@@ -27,7 +27,7 @@ Use the second level heading syntax for the Prerequisites section in the assembl
 [discrete]
 == Assembly Modules
 
-List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly.
+List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly. You can use a `leveloffset` attribute, such as `module-file.adoc[leveloffset=+1]` or `module-file.adoc[leveloffset=+2]`, to set the hierarchy of the module relative to the assembly.
```

Comment:
```suggestion
List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly. You can use a `leveloffset` attribute to set the hierarchy of the module relative to the assembly. In most cases, create modules whose title is an H1 and include those modules by using `[leveloffset=+1]`. Otherwise, if necessary, you can use other leveloffsets values, to promote or demote your module content. 
```
This is one way we could recommend using `[leveloffset=+1]` and make it clear that other offsets are acceptable.

Reply:
+1 to these suggested changes. Still, I'd argue for mandating the use of H1 to start all modules. I see no reason to not use it ever.

Reply:
@rolfedh @rkratky I pushed a commit with a slightly tweaked version of Rolfe's suggestion, to make the examples clear and to mention specifically that offset of 2 or more is what we're saying is okay. That's the main sticking point that I want to be sure people understand is okay so they're not forcing themselves to create artificial assemblies of just two modules just to avoid a leveloffset of 2, when they think doing so is forbidden or something.

```
You can use a `leveloffset` attribute to set the hierarchy of the module relative to the assembly. In most cases, create modules whose title is an H1 and include those modules by using the format `module-file.adoc[leveloffset=+1]`. Otherwise, if necessary, you can use other `leveloffset` values, such as `module-file.adoc[leveloffset=+2]`, to modify the structure of your module content.
```